tox.ini

```diff
@@ -15,5 +15,7 @@ ignore=
     # Docstring Content
     D400,D401,D402,D404,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414,D416,D417
     # Type Annotations
-    ANN002,ANN003,ANN101,ANN102,ANN204,ANN206
+    ANN002,ANN003,ANN101,ANN102,ANN204,ANN206,
```

Comment:
nit: if adding a comma at the end of this line, should they be added for the other lines? (eg L16)

Reply:
Please see the PR description.